src/sentry/static/sentry/app/views/alerts/details/chart.tsx

```diff
@@ -74,7 +74,7 @@ export default class Chart extends React.PureComponent<Props> {
           {
             // e.g. Events or Users
             seriesName: getDisplayForAlertRuleAggregation(aggregation),
-            dataArray: chartData,
+            dataArray: chartData.splice(-1, 1),
```

Comment:
You probably want slice, which returns a new array where as splice alters the array in place and returns an array of the deleted items

```suggestion
            dataArray: chartData.slice(0, -1),
```